src/app/features/landingpage/views/landing-page/landing-page.component.html

```diff
@@ -1,63 +1,76 @@
-  <p class="text-black">lorem ipsum</p>
-  <a
-    routerLink="/ahb"
-    class="bg-primary text-white font-bold py-2 px-4 mt-3 rounded"
-  >
-    Jetzt oeffnen
-  </a>
+<section class="flex flex-col lg:flex-row bg-tint">
+  <div class="flex-1 px-10 pt-10">
+    <h4 class="font-bold text-sm mb-2">
+      Willkommen bei Ahbesser – Anwendungshandbücher für Menschen
+    </h4>
+    <p class="text-[12px]">
+      Bei uns steht der Mensch im Mittelpunkt. Unser Ziel ist es, die komplexen
+      und oft schwer verständlichen Anwendungshandbücher der deutschen
+      Energiewirtschaft benutzerfreundlich und zugänglich zu machen. Mit unserem
+      Service können Sie gezielt nach FormatVersionen und Prüfidentifikatoren
+      suchen und ein einzelnes Dokument nach spezifischen Details durchsuchen.
+      So sparen Sie wertvolle Zeit und Nerven, die Sie stattdessen in Ihr
+      Kerngeschäft investieren können.
+    </p>
+  </div>
+  <div class="flex-1 px-10 pt-10">
+    <h4 class="font-bold text-sm mb-2">
+      Effiziente Suche und einfache Navigation
+    </h4>
+    <p class="text-[12px]">
+      Wir wissen, dass das Durchforsten riesiger PDF-Dateien eine mühsame und
+      zeitraubende Aufgabe sein kann. Daher bieten wir Ihnen eine innovative
+      Plattform, die diesen Prozess erheblich vereinfacht. Unsere
+      benutzerfreundliche Suchfunktion ermöglicht es Ihnen, schnell und präzise
+      die Informationen zu finden, die Sie benötigen. Geben Sie einfach die
+      gewünschte FormatVersion oder den Prüfidentifikator ein, und schon
+      gelangen Sie zum relevanten Dokument – klar strukturiert und leicht
+      verständlich aufbereitet.
+    </p>
+  </div>
+  <div class="flex-1 px-10 pt-10">
+    <h4 class="font-bold text-sm mb-2">Ihre Vorteile auf einen Blick</h4>
+    <p class="text-[12px]">
+      Mit AHBesser wird die Arbeit mit Anwendungshandbüchern so effizient und
+      unkompliziert wie nie zuvor. Unsere Plattform bietet Ihnen:
+    </p>
+    <ul class="list-disc ps-2 my-2">
+      <li class="text-[12px]">
+        <span class="font-bold">Zeitersparnis:</span> Finden Sie gezielt die
+        Informationen, die Sie benötigen, ohne lange suchen zu müssen.
+      </li>
+      <li class="text-[12px]">
+        <span class="font-bold">Benutzerfreundlichkeit:</span> Eine intuitive
+        Navigation und klare Strukturen erleichtern Ihnen den Zugang zu den
+        Dokumenten.
+      </li>
+      <li class="text-[12px]">
+        <span class="font-bold">Transparenz und Klarheit:</span> Alle relevanten
+        Details auf einen Blick, ohne die Notwendigkeit, sich durch
+        unübersichtliche PDF-Dateien zu kämpfen.
+      </li>
+    </ul>
+    <p class="text-[12px]">
+      Erleben Sie den Unterschied und optimieren Sie Ihre Arbeitsabläufe mit
+      unserer intelligenten Lösung für die Verwaltung von Anwendungshandbüchern
+      in der Energiewirtschaft. Willkommen bei AHbesser – weil wir an den
+      Menschen denken.
```

Comment:
```suggestion
      in der Energiewirtschaft. Willkommen bei AHB Tabellen – weil wir an den
      Menschen denken.
```
es schmerzt schon sehr, vll mache ich das Fass nochmal auf.

Reply:
und was für ein Hammer Slogan! 😆